### DISCUSSION TITLE
Error in Windows 10: The procedure entry point pthread_setname_np could not be found in the dynamic link library.

### DISCUSSION THREAD

**Discussion by Francy2007** [MusicPlayerDaemon/MPD#1795] [Q&A]
Greetings,
I installed Mpd version 0.23.12 on Windows 10 (version 0.23.10 works perfectly). When I start the app I get the error "The procedure entry point pthread_setname_np could not be found in the dynamic link library". Can you tell me how I can solve it?
Best regards.

Vincent

**Comment by MaxKellermann** [comment]
How did you build MPD?

**Reply by Francy2007** [reply]
Good morning,
I created a folder in C and the .conf file is as follows:
-------------------------------------------------- ------------------------
music_directory "C:/Users/wins/Music"
playlist_directory "C:/mpd/playlists"
db_file "C:/mpd/mpd.db"
log_file "C:/mpd/mpd.log"
port "6600"
auto_update "yes"
inputs {
         "local" plugins
}

audio_output {
   type "wasapi"
   name "My WASAPI output"
## device "2-AudioQuest Dragonfly"
   mixer_control "PCM"
   mixer_type "software"
   format "44100:24:2"
   enabled "yes"
   enumerate "no"
   exclusive "no"
   after "no"
}

playlist_plugin {
     name "m3u"
     enabled "yes"
as_directory "yes"
}
decoder {
        enabled "no"
        "wildmidi" plugin
        config_file "/etc/timidity/timidity.cfg"
}
-------------------------------------------------- ------------------------



I launch MPD with the following .bat file:
-------------------------------------------------- -----
C:
cd C:\mpd
mpd mpd.conf
-------------------------------------------------- ----------------

I remember that with version 0.23.10 everything works perfectly.
With the Stylophone app I use MPD.
Best regards.


Vincent

**Reply by MaxKellermann** [reply]
That doesn't answer my question. But maybe you cannot answer it because you did not build it. How did you install MPD?

**Comment by Francy2007** [comment]
Hi Max,
i simplified mpd.conf as follows but i still get the same error. Version 0.23.10 still works.
-------------------------------------------------- -------------------------
music_directory "C:/Users/wins/Music"
playlist_directory "C:/mpd/playlists"
db_file "C:/mpd/mpd.db"
log_file "C:/mpd/mpd.log"
audio_output {
   type "wasapi"
   name "My WASAPI output"
   mixer_type "software"
 
}
decoder {
        enabled "no"
        "wildmidi" plugin
        config_file "/etc/timidity/timidity.cfg"
}
-------------------------------------------------- ------------------------------------------------

Best regards.


Vincent

**Comment by Francy2007** [comment]
Hi Max,
I ask if you could kindly give me a solution for the problem. I specify that the error in question is signaled (equally on 2 PCs with o.s. Windows 10 Pro and Intel I3 processors) after inserting the "libwinpthread-1" dll in the Mpd folder as requested by Windows and after starting mpd.conf .
Best regards.

Vincent

**Reply by MaxKellermann** [reply]
Inserting a DLL is a mistake. At best, this does nothing, but certainly will not solve any problem. Don't do that.

**Comment by Francy2007** [comment]
I hope I answer correctly. As per attached photo of Mpd folder.
Greetings.
![Folder Mpd](https://user-images.githubusercontent.com/103477255/235761308-73316c2e-cca7-401f-82a7-b1cab725390b.png)
